src/SFML/Window/macOS/HIDInputManager.hpp

```diff
@@ -40,7 +41,7 @@
 namespace sf::priv
 {
 
-using IOHIDElements = std::vector<IOHIDElementRef>;
+using IOHIDElements = std::vector<CFPtr<__IOHIDElement>>;
```

Comment:
I don't know much about Obj-C, but should we really be using types that start with `__`?

Reply:
I was hesitant to use types with those names too. Does Apple not want users to directly use such types? I'm not sure. A much more verbose alternative would be `std::remove_pointer_t<IOHIDElementRef>`.

Reply:
I'd probably go with that. Not a fan of using "reserved"/internal symbols